Makefile

```diff
@@ -188,6 +189,46 @@ DEFAULT_DOCKER_IMAGES_RULES = \
 docker-images: $(DEFAULT_DOCKER_IMAGES_RULES) ## Build all docker images
 	@echo Done.
 
+.PHONY: docker-images
+promote-docker-images: ## Take existing release (e.g.: RC) and promote to stable
+	$(eval JUST_RETAG=$(shell echo "base models log-collector"))
+	@echo "Promoting RC $(RC_VERSION) to stable $(STABLE_VERSION)"
+	@for target in $(DEFAULT_DOCKER_IMAGES_RULES); do \
```

Comment:
`DEFAULT_DOCKER_IMAGES_RULES` does not contain python 3.7 images
You can reuse the matrix from the github actions but it requires that you move this logic to the github action

Reply:
🤮 
I will wait for 1.5.0 :)